testing/smoke/lib.sh

```diff
@@ -43,28 +43,25 @@ terraform_init() {
     terraform validate >> tf.log
 }
 
+cleanup_tfvar() {
+    if [[ -f terraform.tfvars ]]; then rm terraform.tfvars; fi
+}
+
 append_tfvar() {
-    if [[ -f terraform.tfvars && ! -z ${3} && ${3} -gt 0 ]]; then rm terraform.tfvars; fi
+    echo "-> Adding tfvar ${1}=\"${2}\""
     echo ${1}=\"${2}\" >> terraform.tfvars
 }
 
 terraform_apply() {
-    echo "-> Creating / Upgrading deployment to version ${1}"
```

Comment:
Some output would still be helpful when checking the logs.

Reply:
Added!